src/utils/models/index.ts

```diff
@@ -55,3 +56,46 @@ export const NetworkOperatorModel: K8sModel = {
   namespaced: false,
   plural: 'networks',
 };
+
+export const UserDefinedNetworkModel: K8sModel = {
+  abbr: 'UDN',
+  apiGroup: 'k8s.ovn.org',
+  apiVersion: 'v1',
+  crd: true,
+  id: 'userdefinednetwork',
+  kind: 'UserDefinedNetwork',
+  label: 'userdefinednetwork',
+  // t('UserDefinedNetwork')
+  labelKey: 'UserDefinedNetwork',
```

Comment:
can you translate directly and not in comment like in NetworkOperatorModel in row 46?